src/api.jl

```diff
@@ -1,6 +1,17 @@
 function quandlget(id::AbstractString; order="des", rows=100, frequency="daily", transformation="none",
                    from="", to="", format="TimeArray", api_key="")
 
+    if api_key==""
+        if !ispath(Pkg.dir("Quandl/token/"))
```

Comment:
This has been updated to read 
` if !ispath(joinpath(dirname(@__FILE__),"../token/"))`